ui/lib/apps/SlowQuery/pages/List/index.tsx

```diff
@@ -67,98 +66,97 @@ function List() {
+
+          <Space>
+            {columns.length > 0 && (
+              <ColumnsSelector
+                columns={columns}
+                visibleColumnKeys={visibleColumnKeys}
+                resetColumnKeys={defSlowQueryColumnKeys}
+                onChange={setVisibleColumnKeys}
+                foot={
+                  <Checkbox
+                    checked={showFullSQL}
+                    onChange={(e) => setShowFullSQL(e.target.checked)}
+                  >
+                    {t(
+                      'statement.pages.overview.toolbar.select_columns.show_full_sql'
+                    )}
+                  </Checkbox>
                 }
               />
-              <Select
-                value={queryOptions.limit}
-                style={{ width: 150 }}
-                onChange={(limit) =>
-                  setQueryOptions({ ...queryOptions, limit })
-                }
-              >
-                {LIMITS.map((item) => (
-                  <Option value={item} key={item}>
-                    Limit {item}
-                  </Option>
-                ))}
-              </Select>
-            </Space>
-
-            <Space>
-              {columns.length > 0 && (
-                <ColumnsSelector
-                  columns={columns}
-                  visibleColumnKeys={visibleColumnKeys}
-                  resetColumnKeys={defSlowQueryColumnKeys}
-                  onChange={setVisibleColumnKeys}
-                  foot={
-                    <Checkbox
-                      checked={showFullSQL}
-                      onChange={(e) => setShowFullSQL(e.target.checked)}
-                    >
-                      {t(
-                        'statement.pages.overview.toolbar.select_columns.show_full_sql'
-                      )}
-                    </Checkbox>
-                  }
-                />
+            )}
+            <Tooltip title={t('statement.pages.overview.toolbar.refresh')}>
+              {loadingSlowQueries ? (
+                <LoadingOutlined />
+              ) : (
+                <ReloadOutlined onClick={refresh} />
               )}
-              <Tooltip title={t('statement.pages.overview.toolbar.refresh')}>
-                {loadingSlowQueries ? (
-                  <LoadingOutlined />
-                ) : (
-                  <ReloadOutlined onClick={refresh} />
-                )}
-              </Tooltip>
-            </Space>
-          </Toolbar>
-        </Card>
-      </Sticky>
+            </Tooltip>
+          </Space>
+        </Toolbar>
+      </Card>
 
-      <SlowQueriesTable
-        loading={loadingSlowQueries}
-        slowQueries={slowQueries}
-        orderBy={orderOptions.orderBy}
-        desc={orderOptions.desc}
-        showFullSQL={showFullSQL}
-        visibleColumnKeys={visibleColumnKeys}
-        onGetColumns={setColumns}
-        onChangeOrder={changeOrder}
-      />
-    </ScrollablePane>
+      <div style={{ height: '100%', position: 'relative' }}>
```

Comment:
will this 100% lead to extra space in the vertical direction?

Reply:
Tested but it does not. This div is required for `ScrollablePane`, it needs a relative position parent, and must set height or max-height for itself or its parent.

Reply:
Let me have a test.

Reply:
Yep, it will lead to some extra space on the page bottom if there are no much data, but I think it is acceptable currently.

![image](https://user-images.githubusercontent.com/1284531/81785220-9b9dcb80-9530-11ea-9095-0b6c28458274.png)